completor/constants.py

```diff
@@ -153,7 +153,7 @@ class _Headers:
     # 11. The length of the valve, lVAL (Scale factor).
     # 12. An integer which determines how the flow scaling factor is calculated.
 
-    # Density Activated Recovery Well Segments (WSEGDAR)
+    # Rate Controlled Production Well Segments (WSEGRCP)
```

Comment:
Change to density driven or something else
```suggestion
    # Density Driven Well Segments (WSEGDENSITY)
```